sty/statements/statements.dtx

```diff
@@ -1368,8 +1368,10 @@ DefConstructor('\inlinedef OptionalKeyVals:omtext {}', sub {
    my $symbolnode = XML::LibXML::Element->new('symbol');
    $symbolnode->setAttribute(name=>$symb);
    $symbolnode->setAttribute("xml:id"=>makeNCName("$symb.def.sym"));
+ if ($statement_ancestor) {
    $statement_ancestor->parentNode->insertBefore($symbolnode,$statement_ancestor);
- }
+  } else {
+   Error('expected', $statement_ancestor, undef, "'$xc' missing ancestor node.");}}
```

Comment:
You could use ```$original_node``` if you need a node, but the important error is missing the ```omdoc:CMP ancestor```, not just an arbitrary ancestor.